protocol/wasmbinding/msg_plugin.go

```diff
@@ -129,6 +149,43 @@ func (m *CustomMessenger) depositToSubaccount(ctx sdk.Context, contractAddr sdk.
 	return nil, nil, nil
 }
 
+func (m *CustomMessenger) withdrawFromSubaccount(ctx sdk.Context, contractAddr sdk.AccAddress, withdrawFromSubaccount *bindings.WithdrawFromSubaccount) ([]sdk.Event, [][]byte, error) {
+	if withdrawFromSubaccount == nil {
+		return nil, nil, wasmvmtypes.InvalidRequest{Err: "withdraw from subaccount null withdraw"}
+	}
+
+	senderAddress, err := parseAddress(withdrawFromSubaccount.Sender.Owner)
+	if err != nil {
+		return nil, nil, err
+	}
```

Comment:
nit: This should probably just be the `contractAddr` for the initial implementation, don't think there's any need for a contract to be able to withdraw from another address w/o additional permissioning etc.